src/index.ts

```diff
@@ -17,6 +20,24 @@ const start = async () => {
 		console.log(`Server listening on :${port}`)
 
 		client.login(ENV.DISCORD_BOTTOKEN)
+
+		const communityCount = await CommunityModel.countDocuments()
+		if (communityCount == 0) {
+			// need to create the first community as none exist, so that the api can be used
+			const community = await CommunityModel.create({
+				name: "FAGC Master Community",
+				contact: client.user?.id
+			})
+			const apikey = await createApikey(community, "master")
+			try {
+				const file = await fs.open("./masterapikey.txt", "w", 0o600)
+				await file.write(apikey)
+				await file.close()
```

Comment:
This could be simplified to `await fs.writeFile("./masterapikey.txt", apikey, { flag: "w", mode: 0o600 })`.